src/CurrenciesSpecification.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Money;
+
+use Money\Currencies\Specification;
+
+/**
+ * Specifies information about a currency.
+ *
+ * @author Frederik Bosch <f.bosch@genkgo.nl>
+ */
+interface CurrenciesSpecification
```

Comment:
I think, that it's better to rename this interface like  `CurrencySpecifications`, isn't it?